pycbc/waveform/compress.py

```diff
@@ -259,7 +262,10 @@ def compress_waveform(htilde, sample_points, tolerance, interpolation,
         comp_amp = amp.take(sample_index)
         comp_phase = phase.take(sample_index)
         # update the vecdiffs and mismatch
-        hdecomp = fd_decompress(comp_amp, comp_phase, sample_points,
+        amp_sample_points = None
+        phase_sample_points = sample_points
+        hdecomp = fd_decompress(comp_amp, comp_phase, 
+            amp_sample_points, phase_sample_points,
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->